specification/attributes/unit_format.md

```diff
@@ -0,0 +1,92 @@
+
+### Data Size Unit Names
+
+Data size MUST be abbreviated using the following standard abbreviations.  Each data size abbreviation can be considered both the singular and plural form of the unit.  For example, "GB" is both the singular and plural form of the unit "GB".  The following table lists the valid data size units.
+
+| Data size in bits    | Data size in bytes    |
+| -------------------- | --------------------- |
+| Kib (kibibit = 2^10) | KiB (kibibyte = 2^10) |
+| Mib (mebibit = 2^20) | MiB (mebibyte = 2^20) |
+| Gib (gibibit = 2^30) | GiB (gibibyte = 2^30) |
+| Tib (tebibit = 2^40) | TiB (tebibyte = 2^40) |
+| Pib (pebibit = 2^50) | PiB (pebibyte = 2^50) |
+| Eib (exbibit = 2^60) | EiB (exbibyte = 2^60) |
+| Kb (kilobit = 10^3)  | KB (kilobyte = 10^3)  |
+| Mb (megabit = 10^6)  | MB (megabyte = 10^6)  |
+| Gb (gigabit = 10^9)  | GB (gigabyte = 10^9)  |
+| Tb (terabit = 10^12) | TB (terabyte = 10^12) |
+| Pb (petabit = 10^15) | PB (petabyte = 10^15) |
+| Eb (exabit = 10^15)  | EB (exabyte = 10^15)  |
+
+### Count-based Unit Names
+
+A count-based unit is a noun that represents a discrete number of items, events, or services.  For example, a count-based unit can be used to represent the number of requests, instances, tokens, or connections.  
+
+If the list of recommended values does not cover a count-based unit, a provider MAY introduce a new noun representing a count-based unit.  All nouns appearing in unit that are not listed in the recommended values table will be considered count-based units.  A new count-based unit value MUST be capitalized.
+
+| Count        |
+|--------------|
+| Count        |
+| Unit         |
+| Request      |
+| Token        |
+| Connection   |
+| Certificate  |
+| Domain       |
+| Core         |
+
+### Time-based Unit Names
+
+A time-based unit is a noun that represents a time interval.  Time-based units can be used to measure consumption over a time interval or in combination with another unit to capture a rate of consumption.
+
+| Time         |
+|--------------|
+| Year         |
+| Month        |
+| Day          |
+| Hour         |
+| Minute       |
+| Second       |
+
+### Composite Units
+
+If the unit value is a composite value made from combinations of one or more units, each component MUST also align with the set of recommended values.
+
+Instead of "per" or "-" to denote a Composite Unit, slash ("/") and space(" ") MUST be used as a common convention.  Count-based units like requests, instances, and tokens SHOULD be expressed using a value listed in the count dimension.  For example, if a usage unit is measured as a rate of requests or instances over a period of time, the unit SHOULD be listed as "count/day" to signify the number of requests per day.
```

Comment:
"Requests" is in the list of recommended values.  Thus:

```suggestion
Instead of "per" or "-" to denote a Composite Unit, slash ("/") and space(" ") MUST be used as a common convention.  Count-based units like requests, instances, and tokens SHOULD be expressed using a value listed in the count dimension.  For example, if a usage unit is measured as a rate of requests or instances over a period of time, the unit SHOULD be listed as "request/day" to signify the number of requests per day.
```